index.html

```diff
@@ -1,65 +1,247 @@
+                            </ul>
+                            <button type="button">See Project</button>
+                        </li>
+
+                        <li class="project">
+                            <h3>Profesional Art<br>Printing Data</h3>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                            <ul class="build-with">
+                                <li>html</li>
+                                <li>bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                            <button type="button">See Project</button>
+                        </li>
+
+                        <li class="project">
+                            <h3>Profesional Art<br>Printing Data</h3>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                            <ul class="build-with">
+                                <li>html</li>
+                                <li>bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                            <button type="button">See Project</button>
+                        </li>
+                    </ul>
+                </div>
+            </section>
+        
+            <!--About Me Section-->
+            <section class="about">
+                <div class="about-me">
+                <h2>About me</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <button type="button" class="btn">Get My Resume</button>
+                </div>
+                <hr>
+                <div class="qualifications">
+                <div class="languages">
+                    <div class="qualifications-title">
+                    <img src="./img/Rectangle-55.png" alt="">
+                    <h3>Language</h3>
+                    </div>
+                    <ul>
+                    <li>Javascript</li>
+                    <li>Ruby</li>
+                    <li>Html</li>
+                    <li>Css</li>
+                    </ul>
+                </div>
+                <div class="frameworks">
+                    <div class="qualifications-title">
+                    <img src="./img/Rectangle-56.png" alt="">
+                    <h3>Frameworks</h3>
+                    </div>
+                    <ul>
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
+                    </ul>
+                </div>
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") should be wrapped in  a `ul`, and  each of them in a `li` , kindly fixt that